File: src/main/java/blockswarm/network/cluster/PeerAddress.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package blockswarm.network.cluster;

import java.net.InetAddress;
import java.net.InetSocketAddress;

/**
 *
 * @author cal
 */
public class PeerAddress {
    InetSocketAddress address;
    
    public PeerAddress(InetSocketAddress address)
    {
        this.address = address;
    }
    
    public InetSocketAddress inetAddress()
    {
        return address;
    }
}

File: src/main/java/blockswarm/network/cluster/PeerAddress.kt
package blockswarm.network.cluster

import java.io.Serializable
import java.net.InetSocketAddress

/**
 *
 * @author cal
 */
class PeerAddress(internal var address: InetSocketAddress) : Serializable {

    fun inetAddress(): InetSocketAddress {
        return address
    }
}
